util.js

```diff
@@ -32,3 +32,19 @@ exports.runSpawn = function runSpawn({ cwd } = {}) {
     })
   }
 }
+
+exports.runSpawn = runSpawn
+
+exports.tagVersionInGit = async function (version) {
+  const run = runSpawn()
+
+  await run('git', ['push', 'origin', `:refs/tags/${version}`])
```

Comment:
what does this do? I think publishing a release already creates a tag. let's make sure this doesn't screw anything up

Reply:
If we had an old `v2` tag, we can't force update it I think. So this line deletes it from the remote and then next line adds it again pointing to the current commit.

Reply:
That's the idea yeah. Running the same code without that line will give you the following error
```
 ! [rejected]        v5 -> v5 (already exists)
error: failed to push some refs to 'https://github.com/ja-bravo/optic-action-test.git'
hint: Updates were rejected because the tag already exists in the remote
```